multi_import/formats.py

```diff
@@ -58,14 +58,14 @@ def detect(self, file_handler, file_contents):
         return False
 
     def pre_read(self, file_object):
-        return file_object
+        return file_object.read()
```

Comment:
What's this method do, and where is it used? It seems weird you are doing a `read` in a method called `pre_read`.

Reply:
Yeah you're right, `pre_read()` is probably not the best place, I'll move it.